### ISSUE TITLE
General Config Tab Crashes

### ISSUE THREAD

**Issue by sylv256** [Siphalor/mouse-wheelie#130]
**Version**
`1.17.1`
`1.7.3-newconfig+mc1.17.1-pre1`
**Describe the bug**
if you click the general config tab button, crash

**To Reproduce**
1. open the general config tab

**Expected behavior**
The general config tab should open

**Log**
```
java.lang.AbstractMethodError: Receiver class de.siphalor.coat.input.TextConfigInput does not define or inherit an implementation of the resolved method 'abstract void tick()' of interface de.siphalor.coat.input.ConfigInput.
	at Not Enough Crashes deobfuscated stack trace.(1.17.1+build.24)
	at de.siphalor.coat.list.entry.ConfigListConfigEntry.tick(ConfigListConfigEntry.java:157)
	at de.siphalor.coat.list.DynamicEntryListWidget.tick(DynamicEntryListWidget.java:641)
	at de.siphalor.coat.screen.ConfigScreen.tick(ConfigScreen.java:246)
```

**Comment by Siphalor** [comment]
Fixed thanks to Siphalor/coat#2

**Comment by Frederick888** [comment]
While 'General' now works, it seems 'Sorting' still has this issue.

**Comment by Siphalor** [comment]
A fix for this is implemented now in Coat and will probably make it into the next beta :)
I'm keeping this open as a reminder to myself.

**Comment by katubug** [comment]
Hi, I just received what I think is the same crash: https://bytebin.lucko.me/BVVfRP70EQ

I got it while accessing General, using mousewheelie-1.7.3-newconfig+mc1.17.1-pre1 - please let me know if you'd like any more info.

**Comment by Siphalor** [comment]
Hello @katubug, these crashes should be fixed in 1.7.3-newconfig.3 and onwards, so just update to the latest versions and this is hopefully fixed already :)

**Comment by katubug** [comment]
Oh gotcha! Sorry, I was on the latest curseforge version. Grabbing the correct one from modrinth now. Thanks!
